src/utils.ts

```diff
@@ -735,6 +735,44 @@ export async function getSpaceController(
   return await getEnsOwner(id, network, options);
 }
 
+export async function getShibariumNameOwner(
+  id: string,
+  network: string
+): Promise<string> {
+  if (!id.endsWith('.shib')) {
+    return EMPTY_ADDRESS;
+  }
+
+  const response = await fetch(
+    `https://bens.services.blockscout.com/api/v1/${network}/domains:lookup?name=${id}&only_active=true`,
```

Comment:
What is different from this service and API use in stamp https://github.com/stamp-labs/stamp/pull/362/files#diff-dbbfcf7aecae7147af6069bcbaabc0db16a1dccfb2878230de0771bea5aed776R7-R8 ?